src/cursor.ts

```diff
@@ -268,17 +279,21 @@ export function defaultEncodeCursor<
   const cursor = new URLSearchParams();
 
   for (const [key, value] of values) {
-    switch (typeof value) {
-      case "string":
+    switch (true) {
```

Comment:
I changed this to support nullables by default in the cursor encoder/decoder. Not sure that's something we want to do? But all current tests pass without modification here so it looks ok I think?